README.md

```diff
@@ -112,3 +112,17 @@ $ gulp watch
 [16:50:20] Finished 'scripts:app' after 454 ms
 
 ```
+
+## Deploying to Cloud Foundry
+To deploy to Cloud Foundry we will assume a environment without internet access.
+First we will have to cache our dependencies:
+```
+rm -r ./node_modules/* npm-shrinkwrap.json # First delete all to get rid of the devDependencies
```

Comment:
You should delete the .cfignore file as well because w/o that CF will not push the node_modules to CF. You can also delete to bower_components folder as well, since after compilation it's not required anymore and can be quite a big overhead while pushing. Additionally, if you remove the shrinkwrap step, there is no need to remove the shrinkwrap.json (see comment below).